opennms-full-assembly/pom.xml

```diff
@@ -615,8 +615,17 @@
                 <!-- Include the OpenNMS Minion features produced by container/features/pom.xml -->
                 <feature>minion-core-api</feature>
                 <feature>opennms-dao-minion</feature>
-                <feature>opennms-deviceconfig</feature>
+                <!-- Lots of deviceconfig features -->
+                <feature>opennms-deviceconfig-deps</feature>
                 <feature>opennms-deviceconfig-rest</feature>
+                <feature>opennms-deviceconfig-tftp</feature>
```

Comment:
Shrink this to one module for sink

Reply:
`tftp` is used both, by `sink` and by `monitor`. Shouldn't we keep it separate and reference it in these features?

Reply:
Don't need `opennms-deviceconfig-monitor` here as it is not `osgi` module on OpenNMS.  Just need `opennms-deviceconfig-sink-consumer`  and `opennms-deviceconfig-sink` here.

Reply:
I wasn't aware that transitive features are included automatically. Great that they can be removed here.

Reply:
Removed `opennms-deviceconfig-monitor` OSGI feature from `opennms-full-assembly`.